app/views/trainees/subject_specialisms/edit.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= render PageTitle::View.new(i18n_key: "trainees.course_details.edit", has_errors: @select_specialism_form.errors.present?) %>
```

Comment:
```suggestion
<%= render PageTitle::View.new(i18n_key: "trainees.subject_specialisms.edit", has_errors: @select_specialism_form.errors.present?) %>
```

(needs translation added)

Reply:
🤦 Forgot to change one thing from the copy pasta